.gitignore

```diff
@@ -16,3 +16,6 @@ node_modules
 .idea/
 *.iml
 *.iws
+
+# Nodenv
+.node-version
```

Comment:
FYI: I've got these ignored globally because they always piss out the `nvm` crowd. There's a `nodenv` plugin that makes it respect the `.nvmrc` files.